src/components/UserInfo.js

```diff
@@ -1,20 +1,29 @@
-export default class UserInfo {
-  constructor({ profileTitle, profileDescription }) {
-    this._name = document.querySelector(profileTitle);
+ export default class UserInfo {
+  constructor({ profileTitle, profileDescription, avatarSelector}) {
+    this.name = document.querySelector(profileTitle);
     this._description = document.querySelector(profileDescription);
+    this._avatarElement = document.querySelector(avatarSelector);
+
+    
   }
 
   getUserInfo() {
     this._userData = {
-      name: this._name.textContent,
+      name: this.name.textContent,
       description: this._description.textContent,
+      avatar: this._avatarElement.src,
     }
     return this._userData
   }
 
-  setUserInfo({ name, description }) {
-    this._name.textContent = name;
+  setUserInfo({ name, description, avatar }) {
+    this.name.textContent = name;
     this._description.textContent = description;
+    this._avatarElement.src = avatar;
```

Comment:
Needs correction: there is no need to update the avatar here as you can use this method after editing the profile, use `updateAvatar` method in index.js for updating avatar